connector_magento_configurable/models/inherit.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import models, fields, api
+
+from odoo.addons.queue_job.job import job
+from odoo.addons.component.core import Component
+
+
+class ProductTemplate(models.Model):
+    _inherit = 'product.template'
+
+    @api.multi
+    @job
+    def delayable_unlink(self):
+        """
+            Allows to delay an unlink job.
+            This is used to delete the orphaned product templates
+        """
+        self.unlink()
```

Comment:
A job should be idempotent: if run twice, should not fail. Here, you should just check if the product doesn't exist, ignore the unlink.

Reply:
Obviously, yes